packages/apollo-language-server/src/graphqlTypes.ts

```diff
@@ -1702,108 +1702,108 @@ export enum ValidationErrorType {
  * Filter options to exclude by client reference ID, client name, and client version.
  */
 export interface ClientInfoFilter {
-  name?: string | null;
   referenceID?: string | null;
+  name?: string | null;
```

Comment:
Aside from running codegen, I don't understand why all the shuffling is happening. It looks benign but a bit surprising. Is it possible the newly imported `withTypenameFieldAddedWhereNeeded` has this effect?

Reply:
@trevor-scheer this was the result of running `client:codegen` which was required due to failing CI tests (caused by changes to our API). I just committed the new resulting `graphqlTypes.ts` file as is, but I agree - it's very annoying that things get re-arranged like this. 😞

Reply:
I just re-ran `npm run client:codegen` again as I was worried property order wasn't being preserved between invocations; luckily the output was the same as the committed file. Maybe we'll leave this as is since it is benign, and it's now lined up if someone runs this command again?